_conferences/devfest-prague-2019.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "DevFest CZ"
```

Comment:
This should be "GDG DevFest" to match the others

Reply:
Okay. We don't use this name, but we can do it here so that it matches the others.